projects/elements/src/lib/lazy-elements/lazy-element/lazy-element.directive.spec.ts

```diff
@@ -1,7 +1,27 @@
-import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import {
+  Component,
+  CUSTOM_ELEMENTS_SCHEMA,
+  ComponentFactoryResolver,
+  ViewContainerRef,
+  NgModule
+} from '@angular/core';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { LazyElementDirective } from './lazy-element.directive';
+import { LazyElementsLoaderService } from '../lazy-elements-loader.service';
+
+@Component({
+  template: `
+    <p class="loading">Spinner...</p>
+  `
+})
+class SpinnerComponent {}
```

Comment:
Why does this exists when there is also `/app/shared/spinner/spinner.component.html`, do we even need the
`/app/shared/spinner/spinner.component.html` if we can get it here inline ?

Reply:
this test is in `elements` project, and the spinner component is in `elements-demo`